tools/cjxl_main.cc

```diff
@@ -147,7 +147,7 @@ struct CompressArgs {
     cmdline->AddOptionValue(
         'e', "effort", "EFFORT",
         "Encoder effort setting. Range: 1 .. 9.\n"
-        "     Default: 7. Higher number is more effort (slower).",
+        "     Default: 7. Higher number is more effort (slower) for smaller size.",
```

Comment:
In lossless, this is broadly the goal, but there is no assurance that higher effort produces smaller file sizes. In lossy, the semantics are different: higher effort aims to more accurately and consistently reach the target distance (while also employing more costly tools).

Reply:
Good point, leave it as it was or change it to "resulting in more detail or smaller size"?

Reply:
```suggestion
        "    Default: 7. Higher numbers allow more computation, generally producing smaller files at same quality. For lossy output, higher effort should more accurately reach the target quality.",
```

Reply:
Main thing bugging me about my suggestion is the word 'files' which may not always be semantically accurate, but I'm out of time right now and think it communicates the point to the average user.

Reply:
I think that'll do, this _is_ for cjxl itself rather than the library after all, so it'll probably always be files anyway.